Fix Tombo for use with rpy2 v3 by initialising the DataFrame with a dict
https://github.com/nanoporetech/tombo/issues/208
Patch prepared for EasyBuild by Simon Branford, University of Birmingham
--- tombo/_plot_commands.py.orig	2020-12-16 14:28:04.891944000 +0000
+++ tombo/_plot_commands.py	2020-12-16 14:28:14.682304346 +0000
@@ -44,7 +44,7 @@
 try:
     from rpy2 import robjects as r
     from rpy2.robjects.packages import importr
-    _R_DF = r.DataFrame(())
+    _R_DF = r.DataFrame({})
 except:
     # pass here and print detailed error when main functions are actually called
     # in  to give specific error message
